src/plugins/intel_npu/src/al/include/intel_npu/npuw_private_properties.hpp

```diff
@@ -412,6 +412,14 @@ static constexpr ov::Property<uint32_t> max_prompt_len{"NPUW_LLM_MAX_PROMPT_LEN"
  */
 static constexpr ov::Property<uint32_t> min_response_len{"NPUW_LLM_MIN_RESPONSE_LEN"};
 
+/**
+ * @brief
+ * Type: std::map<std::string, ov::Any>.
+ * Tell NPUW the configuration for compilation of prefill model.
+ * NOTE: !! Write-only !!
+ */
+static constexpr ov::Property<std::string> prefill_config{"NPUW_LLM_PREFILL_CONFIG"};
```

Comment:
Wondering why do we even need the `Property` for this? 

They idea is that user may provide it like this:
```
model = read_model(...);
auto compiled = core.compile_model(model, "NPU", { "NPUW_LLM_PREFILL_CONFIG": {...} });
```

Note, there is no need for user to `set` or `get` this config later on. It just should be passed once

Reply:
It is just for us that all things are in one place

Reply:
Plus these are also properties, they shouldn't be handled another way, because it will seem as hack. We need unified place to show all properties we have and unified way of handling them.

Reply:
> It is just for us that all things are in one place

TBH, didn't get the point. What are the `things` and why there should be in one place?

My point is that having llm config params (e.g `NPUW_LLM_PREFILL_CONFIG`, ...) as `properties` complicates implementation as it brings more responsibilities to properly handle them. When it's just `ov::AnyMap`, it's parsed once in `llm_compiled_model.cpp` and then forgotten.

Reply:
All properties that are passed into `core.compile_model(model, device, properties)` should be handled in one place.
It is passed as `properties` into `core.compile_model(...)`, not as any `extra` parameters or some custom thing. That means we should treat this as property.